server/neptune/workflows/internal/activities/main.go

```diff
@@ -37,13 +52,46 @@ type Terraform struct {
 	*cleanupActivities
 }
 
-func NewTerraform(serverURL *url.URL) *Terraform {
+func NewTerraform(config neptune.TerraformConfig, dataDir string, scope tally.Scope, serverURL *url.URL) (*Terraform, error) {
+	binDir, err := mkSubDir(dataDir, BinDirName)
+	if err != nil {
+		return nil, err
+	}
+
+	cacheDir, err := mkSubDir(dataDir, TerraformPluginCacheDirName)
+	if err != nil {
+		return nil, err
+	}
+
+	defaultTfVersion, err := version.NewVersion(config.DefaultVersionStr)
+	if err != nil {
+		return nil, errors.Wrapf(err, "parsing version %s", config.DefaultVersionStr)
+	}
+
+	tfClient, err := terraform.NewAsyncClient(
+		binDir,
+		cacheDir,
+		config.DefaultVersionStr,
+		config.DefaultVersionFlagName,
+		config.DownloadURL,
+		&legacy_tf.DefaultDownloader{},
+		true,
```

Comment:
We currently hardcode this when instantiating the terraform client in the legacy worker: https://github.com/lyft/atlantis/blob/1d8476419cbe01b736c97007d33eb439095f5505/server/server.go#L410-L419 
We could make this configurable but I don't see much value in it since disabling the plugin cache would mean downloading plugins for every root whenever we need to run a terraform command, which would also significantly spike our disk usage imo. 

I think we could just set the `terraformPluginCacheDir` in the command builder and avoid enabling/disabling the plugin cache for neptune. What do you think?

Reply:
We can leave as is to unblock, it was more of a comment on how we are passing in a hardcoded boolean. Seems like we don't ever want the situation for it to be false, so if it is not a configuration, maybe we can get rid of that parameter and just make this the only supported behavior in the future.